File: cpe49題/10221 Satellites.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main(){
	double s,a;
	string min_deg;
	while(cin>>s>>a>>min_deg){
		if(min_deg=="min"){
			a/=60;
		}
		if(a>180){
			a=a-180;
		}
		double ang=a*(M_PI/180);
		double arc=(6440+s)*ang;
		double chord=(2*((6440+s)*sin(ang/2)));
		cout<<fixed<<setprecision(6)<<arc<<" "<<chord<<endl;
	}
}

File: cpe49題/10221 Satellites.py
import math

while True:
	try:
		s,a,c=input().split()
	except EOFError:
		break

	s=int(s)
	a=int(a)

	if c=="min":
		a/=60

	if a > 180:
		a=360-a
		
	r=6440+s
	arc=2*r*math.pi*(a/360)
	
	ang=a*(math.pi/180)
	chord=r*(math.sin(ang/2))*2
	print(f"{arc:.6f} {chord:.6f}")
